src/perf_counters.cc

```diff
@@ -0,0 +1,114 @@
+#include "perf_counters.h"
+
+#if defined HAVE_LIBPFM
+#include <cstring>
```

Comment:
sorry, i just spotted these includes (string, vector) would be needed if `HAVE_LIBPFM` is not defined too.

Reply:
done.